partials/appMainNav.vue

```diff
@@ -43,7 +43,25 @@
       >
         Faucet
       </AppNavLink>
+      <div class="footer mobile">
+        <a
+          :href="apiDocs"
+          target="_blank"
+        >
+          æternity middleware docs
+        </a>
+        <p>{{ mdwVersion }}</p>
+      </div>
     </AppNavAccordion>
+    <div class="footer desktop">
+      <a
+        :href="apiDocs"
+        target="_blank"
+      >
+        æternity middleware docs
+      </a>
+      <p>{{ mdwVersion }}</p>
+    </div>
```

Comment:
```suggestion
      <div class="footer mobile">
        <a
          :href="apiDocs"
          target="_blank"
        >
          API docs
        </a>
        <br>
        {{ mdwVersion }}
      </div>
    </AppNavAccordion>
    <div class="footer desktop">
      <a
        :href="apiDocs"
        target="_blank"
      >
        API docs
      </a>
      <br>
      {{ mdwVersion }}
    </div>
```

Reply:
I like it more like this :D

Reply:
![image](https://user-images.githubusercontent.com/8124114/137490385-477c070c-beeb-447f-b515-5af988a4f64f.png)

Reply:
Yes, its' much better :)